packages/components/src/focal-point-picker/index.js

```diff
@@ -9,62 +10,86 @@ import classnames from 'classnames';
 import { __ } from '@wordpress/i18n';
 import { Component, createRef } from '@wordpress/element';
 import { withInstanceId, compose } from '@wordpress/compose';
-import { Path, SVG } from '@wordpress/primitives';
+import { UP, DOWN, LEFT, RIGHT } from '@wordpress/keycodes';
 
 /**
  * Internal dependencies
  */
-import BaseControl from '../base-control';
 import withFocusOutside from '../higher-order/with-focus-outside';
-
-const TEXTCONTROL_MIN = 0;
-const TEXTCONTROL_MAX = 100;
+import BaseControl from '../base-control';
+import Controls from './controls';
+import FocalPoint from './focal-point';
+import Grid from './grid';
+import Media from './media';
+import {
+	MediaWrapper,
+	MediaContainer,
+} from './styles/focal-point-picker-style';
+import { roundClamp } from '../utils/math';
+import { INITIAL_BOUNDS } from './utils';
 
 export class FocalPointPicker extends Component {
-	constructor( props ) {
-		super( props );
-		this.onMouseMove = this.onMouseMove.bind( this );
+	constructor( props, context ) {
+		super( props, context );
```

Comment:
in general we used to use this pattern `super( ...arguments )` but it's the same thing.

Reply:
That makes sense to me 👍